Source: John
Upstream: No
Reason: The include path without phonon at the end seems to be what most packages (e.g. texstudio) expect

--- CMakeLists.txt	2019-08-27 15:26:52.000000000 +0200
+++ -	2019-08-28 19:26:11.902354706 +0200
@@ -170,7 +170,7 @@
    LIB_NAME ${PHONON_LIB_SONAME}
    DEPS "core widgets"
    FILENAME_VAR pri_filename
-   INCLUDE_INSTALL_DIR ${INCLUDE_INSTALL_DIR}/phonon
+   INCLUDE_INSTALL_DIR ${INCLUDE_INSTALL_DIR}
 )
 install(FILES ${pri_filename} DESTINATION ${ECM_MKSPECS_INSTALL_DIR})
 
